homework-submission/W2-homework/todo-app/src/Components/AddItem/AddItem.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import './AddItem.css';
+
+/**
+ * A class component to add an item to the todo list.
+ * @return This class returns a form to add a todo item and its deadline
+ */
+
+class AddItem extends Component {
+  state = {
+    description: '',
+    deadline: '',
+  };
+
+  handleChange = e => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    if (e.target.description.value === '') {
+      return false;
+    } else {
```

Comment:
If you are returning no need for the `else` close!

the return keyword stops the function in it's exact place and never execute any line after.

Reply:
Thanks... done!